conf/configure.fv3.hera.intel

```diff
@@ -31,10 +31,17 @@ OPENMP = Y
 AVX2 = Y
 HYDRO = N
 CCPP = N
+CUST_POST = Y
+NO_INLINE_POST =
 
 include       $(ESMFMKFILE)
 ESMF_INC    = $(ESMF_F90COMPILEPATHS)
 
+ifeq ($(CUST_POST),Y)
```

Comment:
not sure if we want to have personal customized code in the configure file. This could be kept in a personal branch.

Reply:
I will remove this if block as well.

Reply:
Is cust_post going to remain or not?